metrics/statsd.go

```diff
@@ -29,29 +29,30 @@ func NewStatsdMonitor(client statsdClient, prefix string) *StatsdMonitor {
 }
 
 // RecordRun records a run behavior
-func (m *StatsdMonitor) RecordRun(recipe recipe.Recipe, duration int, success bool) {
+func (m *StatsdMonitor) RecordRun(recipe recipe.Recipe, duration int, success bool, dataCount int) {
 	m.client.Timing(
-		m.createMetricName(runDurationMetricName, recipe, success),
+		m.createMetricName(runDurationMetricName, recipe, success, dataCount),
 		int64(duration),
 	)
 	m.client.Increment(
-		m.createMetricName(runMetricName, recipe, success),
+		m.createMetricName(runMetricName, recipe, success, dataCount),
 	)
 }
 
 // createMetricName creates a metric name for a given recipe and success
-func (m *StatsdMonitor) createMetricName(metricName string, recipe recipe.Recipe, success bool) string {
+func (m *StatsdMonitor) createMetricName(metricName string, recipe recipe.Recipe, success bool, dataCount int) string {
 	var successText = "false"
 	if success {
 		successText = "true"
 	}
 
 	return fmt.Sprintf(
-		"%s.%s,name=%s,success=%s",
+		"%s.%s,name=%s,success=%s,dataCount=%s",
```

Comment:
use `dataCount=%d` instead for formatting number.

and we can skip the using `strconv.Itoa`.